deps.bzl

```diff
@@ -1833,12 +1765,6 @@ def prysm_deps():
         sum = "h1:ujPKutqRlJtcfWk6toYVYagwra7HQHbXOaS171b4Tg8=",
         version = "v0.0.0-20150330215556-f50fe3d243e1",
     )
-    go_repository(
```

Comment:
this got removed... should this be? need help reviewing all this

Reply:
this is mostly generated config. Pruning unused deps is OK